utils/src/main/java/com/cloud/utils/PropertiesUtil.java

```diff
@@ -193,4 +195,19 @@ public static Properties loadFromFile(final File file)
         return properties;
     }
 
+    public static void persist(Properties properties, File file, String key, String value) throws IOException {
```

Comment:
This can be removed as well. See how the keystore passphrase is persisted, follow that pattern. Hope this helps @nvazquez